bin/git-ensembl

```diff
@@ -570,134 +572,98 @@ sub _cmdline_cfg {
-    'ensembl-compara'     => $root.'ensembl-compara.git',
-    'ensembl-variation'   => $root.'ensembl-variation.git',
-    'ensembl-funcgen'     => $root.'ensembl-funcgen.git',
-    'ensembl-tools'       => $root.'ensembl-tools.git',
-    'ensembl-vep'         => $root.'ensembl-vep.git',
-    'ensembl-io'          => $root.'ensembl-io.git',
-    'ensembl-production'  => $root.'ensembl-production.git',
-    'ensembl-rest'        => $root.'ensembl-rest.git',
-    'ensembl-webcode'     => $root.'ensembl-webcode.git',
-    'ensembl-orm'         => $root.'ensembl-orm.git',
-    'ensembl-hdf5'        => $root.'ensembl-hdf5.git',
-    'public-plugins'      => $root.'public-plugins.git',
-    'ebi-plugins'         => $root.'ebi-plugins.git',
-    'biomart-perl'        => $root.'biomart-perl.git',
-    'web-utils'           => $root.'web-utils.git',
-    'ensembl-hive'        => $root.'ensembl-hive.git',
-    'ensembl-analysis'    => $root.'ensembl-analysis.git',
-    'ensembl-pipeline'    => $root.'ensembl-pipeline.git',
-    'ensembl-killlist'    => $root.'ensembl-killlist.git',
-    'ensembl-test'        => $root.'ensembl-test.git',
-    'ensj-healthcheck'    => $root.'ensj-healthcheck.git',
-    'WiggleTools'         => $root.'WiggleTools.git',
-    'VersioningService'   => $root.'VersioningService.git',
-    'ensembl-doc'         => $root.'ensembl-doc.git',
-    'ensembl-xs'          => $root.'ensembl-xs.git',
-    'ensembl-internal-variation'   => $root.'ensembl-internal-variation.git',
-    'ensembl-test'        => $root.'ensembl-test.git',
-    'ensembl-taxonomy'    => $root.'ensembl-taxonomy.git',
-    'ensembl-git-tools'   => $root.'ensembl-git-tools.git',
-    'ensembl-metadata'    => $root.'ensembl-metadata.git',
-    'VEP_plugins'         => $root.'VEP_plugins.git',
-    'eg-web-common'       => $eg_root.'eg-web-common.git',
-    'eg-web-search'       => $eg_root.'eg-web-search.git',
-    'eg-web-bacteria'     => $eg_root.'eg-web-bacteria.git',
-    'eg-web-fungi'        => $eg_root.'eg-web-fungi.git',
-    'eg-web-metazoa'      => $eg_root.'eg-web-metazoa.git',
-    'eg-web-plants'       => $eg_root.'eg-web-plants.git',
-    'eg-web-protists'     => $eg_root.'eg-web-protists.git',
-    'eg-web-g1k'          => $eg_root.'eg-web-g1k.git',
-    'eg-web-parasite'     => $eg_root.'eg-web-parasite.git',
-    'eg-web-testsuite'    => $eg_root.'eg-web-testsuite.git',
-    'eg-web-vectorbase'   => $eg_root.'eg-web-vectorbase.git',
-    'eg-rest'             => $eg_root.'eg-rest.git',
-    'ensemblgenomes-api'  => $eg_root.'ensemblgenomes-api.git',
-    'eg-web-ensembl-configs'  => $eg_root.'eg-web-ensembl-configs.git' 
+  my %def_modules;
+  my $def_groups = _default_groups($opts);
+  foreach my $outter (keys %{$def_groups}) {
+    foreach my $module (@{$def_groups->{$outter}->{modules}}) {
+      if ($module =~ m/^(eg-|ensemblgenomes-)/) {
+        $def_modules{$module} = $eg_root . $module . '.git';
```

Comment:
File::Spec::catfile would be safer to use here - no need to worry about forgetting trailing slashes.

Reply:
Well, let's deal with this another time :-)